packages/zpaq/build.sh

```diff
@@ -0,0 +1,41 @@
+TERMUX_PKG_HOMEPAGE=http://mattmahoney.net/dc/zpaq.html
+TERMUX_PKG_DESCRIPTION="a ultra-compression program"
+TERMUX_PKG_MAINTAINER="Matt Mahoney"
```

Comment:
If you have the intention of updating the termux package and (if there are any) fixing bugs related to Termux, yes put your name in it. Preferably `First_name Last_name @github_nick`. 

Otherwise you can also remove that line completely. It will then default to @fornwall.